src/zap/zapcode.cpp

```diff
@@ -1202,7 +1202,7 @@ void ZapUnwindData::Save(ZapWriter * pZapWriter)
 #endif //REDHAWK
 }
 
-#elif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
+#elif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_) || (defined(_TARGET_X86_) && defined(FEATURE_PAL))
```

Comment:
This ifdef stay as is - we should not have this kind of unwind info for x86.

This is part of the x86 managed code ABI (described in https://github.com/dotnet/coreclr/blob/master/Documentation/botr/clr-abi.md). The  x86 managed code ABI should stay same between Windows and Unix. Otherwise, the x86 Linux port will turn into a very long and complex project.